module4/lessons/Week3/CoffeeChatPD.md

```diff
@@ -0,0 +1,168 @@
+---
+layout: page
+title: Coffee Chats
+---
+
+## Learning Goals
+* Understand what a coffee chat is and why it's incredibly useful
+* Develop a plan for before, during, and after a coffee chat
+
+## What is a Coffee Chat?
+
+A `coffee chat`, also commonly referred to as an `informational interview`, is a casual and informative conversation between two individuals, usually in a relaxed setting like a coffee shop or over zoom and lasting roughly 30 minutes. The main goal of a coffee chat is for one person, often referred to as the "interviewer," to gather insights, advice, and information from the other person, known as the "interviewee," about their job or career.
+
+You will be the "interviewer" in your coffee chat, and you will be interviewing professional software developers about their day-to-day work and career.
+
+<section class="call-to-action" markdown="1">
+With your partner, brainstorm why a coffee chat might be useful for you and where you are in your software development career.
+</section>
+
+<section class="answer" markdown="1">
+### Some Benefits of Coffee Chats

```

Comment:
This header is broken. It displays the `###` instead of an `<h3>` level heading.